invenio_records_rest/views.py

```diff
@@ -64,42 +64,39 @@ def elasticsearch_query_parsing_exception_handler(error):
     return InvalidQueryRESTError(description=description).get_response()
 
 
-elasticsearch_error_handlers = {
```

Comment:
@jirikuncar I think I found a middle-ground which keeps compatibility with `invenio-deposit`: doing the `obj_or_import_string` inside the `init_config`.